lib/api/compiler/class-options.php

```diff
@@ -71,20 +71,21 @@ public function register() {
 						array(
 							'id'      => 'beans_compile_all_scripts',
 							'type'    => 'activation',
+							'label'   => __( 'Select to compile scripts.', 'tm-beans' ),
 							'default' => false,
 						),
 						array(
-							'id'         => 'beans_compile_all_scripts_mode',
-							'type'       => 'select',
-							'default'    => 'aggressive',
-							'attributes' => array( 'style' => 'margin: -3px 0 0 -8px;' ),
-							'options'    => array(
+							'id'      => 'beans_compile_all_scripts_mode',
+							'type'    => 'select',
+							'label'   => 'Choose the level of compilation.',
```

Comment:
Wrap this string literal in `__( 'Choose the level of compilation.' )` for internationalization.